### PR TITLE
feat: Implement shareable package-json-lint config.

### PR DESCRIPTION
**Pull request by yeldiRium**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - 'master'
```
Comment (goloroden) [inline .github/workflows/release.yml:6]: Shouldn't this be `main`?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      "name": "Golo Roden",
+      "email": "golo.roden@thenativeweb.io"
+    },
+    {
+      "name": "Hannes Leutloff",
+      "email": "hannes.leutloff@thenativeweb.io"
+    }
+  ],
+  "private": false,
+  "main": "lib.json",
```
Comment (goloroden) [inline package.json:16]: I'm not sure, but shouldn't this be `./lib.json`?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+## Installation
+
+```shell
+$ npm install npm-package-json-lint-config-tnw \
+              npm-package-json-lint
+```
+
+## Quick Start
+
+This module contains a very strict package-json-lint configuration, one for apps and one for libraries. Its aims to eradicate _any_ variation in your package.json files. It intends to make your package manifests look uniform, enforce that certain fields exist and prevent you from accidentally installing non-strict dependencies. This helps you to narrow down your focus to pure content, as you do not have to think about the order of fields or which fields you need any more. There is very little difference between the config variant for apps and the one for libraries; The difference is mainly that libraries expose an entry point for code imports, while apps expose an executable.
```
Comment (goloroden) [inline README.md:24]:
```suggestion
This module contains a very strict package-json-lint configuration, one for apps and one for libraries. It aims to eradicate _any_ variation in your `package.json` files. It intends to make your package manifests look uniform, enforce that certain fields exist and prevent you from accidentally installing non-strict dependencies. This helps you to narrow down your focus to pure content, as you do not have to think about the order of fields or which fields you need any more. There is very little difference between the config variant for apps and the one for libraries; The difference is mainly that libraries expose an entry point for code imports, while apps expose an executable.
```

File: README.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+To use one of those configurations, create a `.npmpackagejsonlintrc.json` file in your project and use the `extends` keyword.
```
Comment (goloroden) [inline README.md:26]:
```suggestion
To use one of those configurations, create a `.npmpackagejsonlintrc.json` file in your project and use the `extends` keyword:
```

**Review by goloroden — CHANGES_REQUESTED** [review]
Hey @yeldiRium 👋 

This looks pretty good, I have just left a few comments 😊

File: package.json (hunk 2 above)
Comment (yeldiRium) [reply package.json:16]: We usually don't include the `./` part in other modules. It's mostly `build/index.js` or something along the lines.

File: .github/workflows/release.yml (hunk 1 above)
Comment (yeldiRium) [reply .github/workflows/release.yml]: Yep, i copied the actions from an outdated version of another package and forgot to adjust this.

**Review by goloroden — APPROVED** [review]

**Pull request merged**